comment-service/.project

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think these comment-service files ended here by mistake.
There isn't a .gitignore file in here(I removed it), one thing could be to add the whole default gitignore which you have now in the image service, to the root folders ignore file . And add it so that it applies to all files under it. 
So there would be something like `*/.project` -> ignore every .project file in folders under this, not 100% sure how it goes. 
This way all projects would have same ignores.

Reply:
Yeah sure, that would work. Which particular files are usually ignored? `.project`, `.classpath` and everything under `.settings`?

Reply:
Yeh at least those I think.
https://github.com/github/gitignore/blob/master/Maven.gitignore
https://gist.github.com/lordofthelake/5833336

Reply:
Done! Re-review and we're good to go